oshi-core/src/main/java/oshi/software/os/windows/WindowsFileSystem.java

```diff
@@ -281,4 +281,9 @@ public long getOpenFileDescriptors() {
     public long getMaxFileDescriptors() {
         return MAX_WINDOWS_HANDLES;
     }
+
+    @Override
+    public long getMaxFileDescriptorsPerProcess() {
+        return MAX_WINDOWS_HANDLES;
```

Comment:
This is a smaller constant: 16 * 1024 * 1024 (64 bit) or 16 * 1024 * 512 (32 bit).

Reply:
Hmm, I thought that the per-process limit cannot be higher than the value returned by `getMaxFileDescriptors()`. So, wouldn't it make sense to set it like that, because there exists no per-process file limit?

Reply:
Huh, nevermind. These are the same limit, for some reason I thought the systemwide was much larger.

I've seen a per-process of 512 when using the MS C runtime, but that probably doesn't apply here.